From 9e13425377de9100518cf099db002b887ff93ed1 Mon Sep 17 00:00:00 2001
From: Paul Gerber <Paul.Gerber@tq-group.com>
Date: Tue, 25 Mar 2025 13:55:20 +0100
Subject: [PATCH 1/1] tqma8: set CONFIG_SYS_BOOTM_LEN to 64M

The default value is 8M which is too small if a fitimage with initramfs
is booted

Signed-off-by: Paul Gerber <Paul.Gerber@tq-group.com>
---
 include/configs/tqma8mpxl.h | 3 +++
 include/configs/tqma8mx.h   | 3 +++
 include/configs/tqma8mxml.h | 3 +++
 include/configs/tqma8x.h    | 3 +++
 include/configs/tqma8xx.h   | 3 +++
 include/configs/tqma8xxs.h  | 3 +++
 6 files changed, 18 insertions(+)

diff --git a/include/configs/tqma8mpxl.h b/include/configs/tqma8mpxl.h
index 6e832128dbb..4c5743d1cfd 100644
--- a/include/configs/tqma8mpxl.h
+++ b/include/configs/tqma8mpxl.h
@@ -12,6 +12,9 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN			SZ_64M
+
 #define CONFIG_SPL_MAX_SIZE			(152 * 1024)
 #define CONFIG_SYS_MONITOR_LEN			(512 * 1024)
 #define CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR
diff --git a/include/configs/tqma8mx.h b/include/configs/tqma8mx.h
index db48c404889..dcc38fb5b43 100644
--- a/include/configs/tqma8mx.h
+++ b/include/configs/tqma8mx.h
@@ -12,6 +12,9 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN		SZ_64M
+
 #define CONFIG_SPL_MAX_SIZE		(148 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		(512 * 1024)
 
diff --git a/include/configs/tqma8mxml.h b/include/configs/tqma8mxml.h
index 647b177cb3c..01e99dea010 100644
--- a/include/configs/tqma8mxml.h
+++ b/include/configs/tqma8mxml.h
@@ -12,6 +12,9 @@
 #include <asm/arch/imx-regs.h>
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN		SZ_64M
+
 #define CONFIG_SPL_MAX_SIZE		(208 * 1024)
 #define CONFIG_SYS_MONITOR_LEN		SZ_512K
 
diff --git a/include/configs/tqma8x.h b/include/configs/tqma8x.h
index c8ca5cc3b68..7f2a8b4a1ac 100644
--- a/include/configs/tqma8x.h
+++ b/include/configs/tqma8x.h
@@ -13,6 +13,9 @@
 
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN	SZ_64M
+
 /* Display configs */
 /* activate only HDP_LOAD because display port needs firmware,
  * but setting this Option in Kconfig need CONFIG_VIDEO.
diff --git a/include/configs/tqma8xx.h b/include/configs/tqma8xx.h
index c9d4483a126..e1ce6af8404 100644
--- a/include/configs/tqma8xx.h
+++ b/include/configs/tqma8xx.h
@@ -13,6 +13,9 @@
 
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN	SZ_64M
+
 #ifdef CONFIG_SPL_BUILD
 #define CONFIG_PARSE_CONTAINER
 #define CONFIG_SPL_MAX_SIZE				(192 * 1024)
diff --git a/include/configs/tqma8xxs.h b/include/configs/tqma8xxs.h
index dcf2e3802e6..1beb5e83aeb 100644
--- a/include/configs/tqma8xxs.h
+++ b/include/configs/tqma8xxs.h
@@ -13,6 +13,9 @@
 
 #include "imx_env.h"
 
+/* FIT support */
+#define CONFIG_SYS_BOOTM_LEN	SZ_64M
+
 #ifdef CONFIG_SPL_BUILD
 #define CONFIG_PARSE_CONTAINER
 #define CONFIG_SPL_MAX_SIZE				(192 * 1024)
-- 
2.47.2

